charts/dvpe-deployment-gloo/templates/deployment.yaml

```diff
@@ -108,6 +108,16 @@ spec:
             timeoutSeconds: {{ .Values.deployment.spec.containers.readinessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.deployment.spec.containers.readinessProbe.successThreshold }}
             failureThreshold: {{ .Values.deployment.spec.containers.readinessProbe.failureThreshold }}
+          livenessProbe:
```

Comment:
Since liveness probes can cause unexpected application behavior when not properly configured, please make this an optional configuration option. Also, please disable this new feature by default, because all other projects using this chart won't necessarily need a liveness probe in their deployments and changes should be made non invasively without damaging other deployments.

We have to deal with the following issues here:

- If the liveness probe is configured with a low initial delay or period, it may cause the pod to be restarted too frequently, causing the application to be in a state of constant restart and leading to instability.

- If the liveness probe is configured to check an endpoint that is not ready or not responding correctly, it may cause the pod to be restarted even though the application is running correctly.

Please keep in mind that liveness probes are NOT meant to replace the monitoring of your application.

Reply:
The liveness probe is set now to optional. I know liveness probe can have sideeffects. That's why I added default values as recommendation in values.yaml